contrib/devtools/gen-manpages.py

```diff
@@ -8,12 +8,12 @@
 import tempfile
 
 BINARIES = [
-'src/bitcoind',
-'src/bitcoin-cli',
-'src/bitcoin-tx',
-'src/bitcoin-wallet',
-'src/bitcoin-util',
-'src/qt/bitcoin-qt',
+'build/src/bitcoind',
```

Comment:
Wont this break the usage as described in the readme:
```bash
BUILDDIR=$PWD/build ./contrib/devtools/gen-manpages.py 
/root/ci_scratch/build/build/src/bitcoind not found or not an executable
```

Reply:
> To use this tool with out-of-tree builds set `BUILDDIR`.

Does this comment still apply?
Should I change it to `builddir = os.getenv('BUILDDIR', topdir)` to default to the build dir instead of modifying the `BINARIES` paths?

Reply:
Looks like I stepped into something here - reverted `gen-manpages.py` changes, thanks for the comments.